src/scripts/keyboardNavigation.js

```diff
@@ -50,6 +50,11 @@ const NavigationKeys = ['ArrowLeft', 'ArrowRight', 'ArrowUp', 'ArrowDown'];
  */
 const InteractiveElements = ['INPUT', 'TEXTAREA'];
 
+/**
+ * Types of INPUT element for which navigation shouldn't be constrained.
+ */
+const NonInteractiveInputElements = ['button', 'checkbox', 'color', 'file', 'hidden', 'image', 'radio', 'reset', 'submit'];
```

Comment:
Or maybe go the opposite way and list "text" fields? :thinking:

Reply:
It does seem like that would make for a much shorter list to check. :sweat_smile: